packages/core/src/utils/assert-sign-in-experience.ts

```diff
@@ -0,0 +1,26 @@
+import { Branding, BrandingStyle, TermsOfUse } from '@logto/schemas';
+
+import assertThat from '@/utils/assert-that';
+
+export const assertBranding = (branding?: Branding) => {
+  if (!branding) {
+    return;
+  }
+
+  if (branding.style === BrandingStyle.Logo_Slogan) {
```

Comment:
entry condition can also be put into `assertThat()`.

Reply:
Ditto